src/main/java/ladder/view/InputView.java

```diff
@@ -0,0 +1,57 @@
+package ladder.view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final String COMMA = ",";
+
+    private final Scanner scanner = new Scanner(System.in);
+
+    public List<String> readUserNames() {
+        System.out.println("참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)");
+        String input = scanner.nextLine();
+        validateInput(input);
+        return Arrays.stream(input.split(COMMA))
+                .toList();
+    }
+
+    public int readLadderHeight() {
+        System.out.println("\n최대 사다리 높이는 몇 개인가요?");
+        String input = scanner.nextLine();
+        return convertInteger(input);
+
+    }
```

Comment:
```suggestion
        return convertInteger(input);
    }
```

[의견] 불필요한 공백줄은 제거해주세요.